core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/CoarseMesosSchedulerBackend.scala

```diff
@@ -142,8 +154,8 @@ private[spark] class CoarseMesosSchedulerBackend(
       command.setValue(
         s"cd $basename*; $prefixEnv " +
          "./bin/spark-class org.apache.spark.executor.CoarseGrainedExecutorBackend" +
-        s" --driver-url $driverUrl" +
-        s" --executor-id ${offer.getSlaveId.getValue}" +
+        s" --driver-url $driverURL" +
+        s" --executor-id ${sparkExecutorId(offer.getSlaveId.getValue, taskId.toString)}" +
```

Comment:
minor: can you put this in a variable `val executorId = sparkExecutorId(...)` and use it here? For readability